### PR TITLE
Distribution package only including firebase_admin

### PR DESCRIPTION
**Pull request by fgmacedo**
*firebase-admin* is polluting the Python's default package namespace with a *integration* package.

We already have an internal app using *integration* namespace, so the names are in conflict.

I think that *integration* should not be included on the distribution package, since It's related to integration tests only.

This PR changes the pattern for discovering packages to `firebase*`.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -55,7 +55,7 @@
     license=about['__license__'],
     keywords='firebase cloud development',
     install_requires=install_requires,
-    packages=find_packages(exclude=['tests']),
+    packages=find_packages(include=['firebase*']),
```
Comment (hiranya911) [inline setup.py]: Thanks for pointing this out @fgmacedo. Can we drop the wildcard and explicitly make it `['firebase_admin']`?  We don't want anything else exposed from our binaries.

**Comment by fgmacedo** [comment]
@hiranya911 thanks for merging!  Do you have any plans for a new release coming?  We are unable to install the project. Even our fork is not working, because a `pip install -e <fork-url>` clones the project with a high priority path on the `sys.modules` that includes the `integration` module, so the conflict persists. We need a new version on the pypi to be able to install as a source dist.

**Comment by hiranya911** [comment]
@fgmacedo will try to get something out next week.

**Comment by fgmacedo** [comment]
Thanks @hiranya911! :heart:

**Pull request merged**